### PR TITLE
FIxed the parse volume function for linux, tests pass on 	Ubuntu 16.0…

### PR DESCRIPTION
**Pull request by nicogranuja**
…4.4 LTS.

paclt version: 8.0

### CODE REVIEW

**Comment by itchyny** [comment]
Please paste the result of `LANG=C LC_ALL=C pactl list sinks | grep Volume`.

File: volume_linux.go [hunk 1]
```diff
@@ -39,6 +39,9 @@ func parseVolume(out string) (int, error) {
 			!useAmixer && strings.HasPrefix(s, "Volume: 0:") {
 			volumeStr := volumePattern.FindString(s)
 			return strconv.Atoi(volumeStr[:len(volumeStr)-1])
+		} else if strings.HasPrefix(s, "Volume:") {
```
Comment (itchyny) [inline volume_linux.go]: You can just remove the ` 0:` part three lines above.

Comment (nicogranuja) [reply volume_linux.go]: That is correct, sorry completely missed the ||. I had been meaning to pull request since before you implemented amixer support.

**Comment by nicogranuja** [comment]
Result: of running LANG=C LC_ALL=C pactl list sinks | grep Volume
Volume: front-left: 46530 /  71% / -8.92 dB,   front-right: 46530 /  71% / -8.92 dB
	Base Volume: 6554 /  10% / -60.00 dB

**Comment by itchyny** [comment]
Thank you.

**Pull request merged**